packages/fs-detectors/src/detect-builders.ts

```diff
@@ -135,7 +135,7 @@ export async function detectBuilders(
   );
   const withTag = options.tag ? `@${options.tag}` : '';
   const apiMatches = getApiMatches()
-    .filter(b => !ignoreRuntimes.has(b.use))
+    .filter(b => b.config?.middleware || !ignoreRuntimes.has(b.use))
```

Comment:
<!-- Requirement comments refer to a task requirements that is not being met by the current code. Mention what the requirement is and why the code does not meet it.  -->
**requirement (blocking):** ‏find a different solution

Storybook specifically uses `ignoreRuntimes` to prevent middleware from being used.

Ref: https://github.com/vercel/vercel/blob/main/packages/frameworks/src/frameworks.ts#L1943

Can we find a solution that works for both?